docs/config-cli-lint-mode.md

```diff
@@ -8,22 +8,22 @@ description: Cli lint mode can be list_of_files, project or files
 
 # CLI lint mode
 
-Each linter has a lint mode by default, visible in its MegaLinter documentation ([example](https://megalinter.io/latest/descriptors/repository_trivy/#how-the-linting-is-performed)):
+Each linter is pre-configured to use a default lint mode, which are visible in the MegaLinter documentation ([example](https://megalinter.io/latest/descriptors/repository_trivy/#how-the-linting-is-performed)). The possible values are:
 
-- `list_of_files`: All files are sent in single call to the linter
-- `project`: The linter is called from the root of the project, without specifying any file name
-- `file`: The linter is called once by file (so the performances may not be very good)
+- `list_of_files`: The linter is called only once, and passed a list of all the files to be processed
+- `project`: The linter is responsible to scan all the files in the repository, as no filenames are provided it
+- `file`: The linter is called once per file (which hurts performance)
 
-You can override the CLI_LINT_MODE by using configuration variable for each linter (see [linters documentation](https://megalinter.io/supported-linters/))
+You can override the CLI_LINT_MODE by using a configuration variable for each linter (see [linters documentation](https://megalinter.io/supported-linters/)).
 
-- Linters with `file` default lint mode can not be overridden to `list_of_files`
-- Linters with `project` default lint mode can not be overridden to `list_of_files` or `file`
+- Linters using the `file` lint mode cannot be overridden to use the `list_of_files` lint mode
+- Linters using the `project` lint mode cannot be overridden to use either the `list_of_files` or `file` lint modes.
```

Comment:
Since the default lint mode may have been overridden by the user or by some MegaLinter config that they extend, I would say "that default to the" or "defaulting to the" rather than "using the."

Reply:
@Kurt-von-Laven fixed